### PR TITLE
Updating compose

### PR DESCRIPTION
**Pull request by joseims**
Updating the build script so it uses the Dockerfile from repositories(this only exists in the branch new-compose) please review then too!.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ export KUBECONFIG=/path/to/your/kube/config
 ```bash
 ./build.sh
 ```
+For more information about the build script ```./build -h```
```
Comment (liliarsampaio) [inline README.md]: Better follow the structure in the document. For example use:
```bash
./build -h
```
instead of leaving it inline.

Comment (liliarsampaio) [reply README.md]: Check the code above this line to see how to do it.

File: build.sh [hunk 2]
```diff
@@ -1,8 +1,75 @@
...
-git clone https://github.com/ufcg-lsd/asperathos-visualizer ./visualizer/asperathos-visualizer
-docker build visualizer/ -t asperathos_visualizer
+#!/bin/bash
+
+helpFunction()
+{
+   echo ""
+   echo "Usage: $0 -b BRANCH [-r] [-l] [-h]"
+   echo -e "\t-b BRANCH      Select Branch        The branch you want to clone from (the same branch will be used for all projects). If you use -b with a branch different from the current one the files will be deleted and the new branch will be downloaded"
+   echo -e "\t-r             Reset                If you want to reset the current asperathos files"
```
Comment (liliarsampaio) [inline build.sh]: Which files does this refer to? The config files? All code/files? Specify in order to make it clear.

File: build.sh [hunk 3]
```diff
@@ -1,8 +1,75 @@
...
+   echo -e "\t-l             Local Rebuild        If you want to rebuild using the local files. If you use this flag the other's will be ignored."
```
Comment (liliarsampaio) [inline build.sh]: Replace "other's" with "others"

File: build.sh [hunk 4]
```diff
@@ -1,8 +1,75 @@
...
+    rm -rf asperathos-monitor/
+fi
+
+
+
+
+
+if [ -z "$LOCAL" ]
+then
+    echo "Cloning the branch ${BRANCH}"
```
Comment (liliarsampaio) [inline build.sh]: Just a personal preference, when something is currently happening then I like to finish log sentences with "..." so it gives the feeling it is ongoing.

File: build.sh [hunk 5]
```diff
@@ -1,8 +1,75 @@
...
+    git clone https://github.com/ufcg-lsd/asperathos-controller ./asperathos-controller -b $BRANCH
+    git clone https://github.com/ufcg-lsd/asperathos-monitor ./asperathos-monitor -b $BRANCH
+    git clone https://github.com/ufcg-lsd/asperathos-manager ./asperathos-manager -b $BRANCH
+    git clone https://github.com/ufcg-lsd/asperathos-visualizer ./asperathos-visualizer -b $BRANCH
+fi
+
+echo 'Building local images'
```
Comment (liliarsampaio) [inline build.sh]: Same here "..."

**Review by liliarsampaio — CHANGES_REQUESTED** [review]
Some comments inline but no major issue.

**Review by armstrongmsg — APPROVED** [review]

**Review by liliarsampaio — APPROVED** [review]

**Pull request merged**